.awslocal.env

```diff
@@ -12,14 +12,6 @@ AWS_SIGNATURE_VERSION=v4
 
 AWS_MSG_BUCKET_NAME=fsd-notification-bucket
 
-# Import application queue variables
-AWS_SQS_IMPORT_APP_QUEUE_NAME=import-queue.fifo
-AWS_DLQ_IMPORT_APP_QUEUE_NAME=import-dlq.fifo
-AWS_DLQ_MAX_RECEIVE_COUNT=3
-PRIMARY_QUEUE_URL=http://localstack:4566/000000000000/import-queue.fifo
-DEAD_LETTER_QUEUE_URL=http://localstack:4566/000000000000/import-queue-dlq.fifo
-SQS_RECEIVE_MESSAGE_CYCLE_TIME=3
```

Comment:
This is used by the notification service but looks like there's a default in place, so probably fine to remove still 👍